earthpy/spatial.py

```diff
@@ -73,12 +77,22 @@ def stack_raster_tifs(band_paths, out_path, arr_out=True):
         A list with paths to the bands you wish to stack. Bands
         will be stacked in the order given in this list.
     out_path : string
-        A path with a file name for the output stacked raster 
+        A path with a file name for the output stacked raster
          tif file.
     arr_out : boolean
         A boolean argument to designate what is returned in the stacked
         raster tif output.
 
+    Returns
+    ----------
+    If arr_out keyword is True:
+        tuple: The first value representing the result of src.read() of the stacked array and the second value
+        representing the result of src.profile of the stacked array.
+    If arr_out keyword is False:
+        str : A path with a file name for the output stacked raster tif file.
+
+    TODO: Instead of returning a file path when arr_out=False, consider returning None since the out_path is already
```

Comment:
Good recommendation @mikedorfman I will look into that. You are correct in that the user does know that path. Would you want to see a message like "file created" ? We could have `verbose =` as a boolean? Or would you rather it write the file silently?

Reply:
@lwasser I feel like writing the file silently makes the most sense, although I wouldn't be opposed to a verbose keyword.  If you do this, you can simplify lines 114-124 to be:
`
# Write stacked gtif file
with rio.open(out_path, 'w', **dest_kwargs) as dest:
    stack(sources, dest)

if arr_out == True:
    # Read and return array
    with rio.open(out_path, 'r') as src:
        return(src.read(), src.profile)
return None
`